ckanext/ioos_theme/controllers/feedback.py

```diff
@@ -51,11 +52,17 @@ def index(self, data=None, errors=None, error_summary=None):
         data['feedback'] = feedback or ""
         errors = errors or {}
         error_summary = error_summary or {}
+        site_key = config.get('feedback.site_key') or ""
```

Comment:
You can do config.get('feedback.site_key', '') to provide a default if `feedback.site_key` is unset in the config.